pkg/landscaper/webhook/add.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	lscore "github.com/gardener/landscaper/pkg/apis/core"
+	"github.com/gardener/landscaper/pkg/utils/webhook"
+)
+
+var (
+	// DefaultAddOptions are the default AddOptions for AddToManager.
+	DefaultAddOptions = AddOptions{}
+)
+
+// AddOptions are options to apply when adding the AWS shoot webhook to the manager.
+type AddOptions struct{}
+
+// AddToManager creates a webhook with the given options and adds it to the manager.
+func AddToManager(logger logr.Logger, mgr manager.Manager) (*webhook.Webhook, error) {
+	logger.Info("Adding webhook to manager")
+	args := webhook.Args{
+		Name: "landscaper-validation-webhook",
+		Validators: map[webhook.Validator][]runtime.Object{
+			NewValidator(logger.WithName("validator")): {
```

Comment:
the logger with name `validator` is now added 2 times. 
here and in `cmd/landscaper-controller/app/app.go`